Library/ViewModels/ActivityUpdateViewModel.swift

```diff
@@ -115,11 +115,14 @@ private func updatePostedString(forActivity activity: Activity) -> NSAttributedS
 
   let mutableString = NSMutableAttributedString(attributedString: attributedString)
 
-  let timeNumber = time.componentsSeparatedByCharactersInSet(decimalCharacterSet).first
+  let timeNumber = time.components(separatedBy: decimalCharacterSet).first
 
-  if let timeRange = mutableString.string.rangeOfString(time), let timeNumber = timeNumber {
-    let timeStartIndex = mutableString.string.startIndex.distanceTo(timeRange.startIndex)
-    let timeNumberStartIndex = time.startIndex.distanceTo(timeNumber.startIndex)
+  if let timeRange = mutableString.string.range(of: time), let timeNumber = timeNumber {
+
+    let timeStartIndex = mutableString.string
+      .distance(from: mutableString.string.startIndex, to: timeRange.lowerBound)
+    let timeNumberStartIndex = mutableString.string
+      .distance(from: time.startIndex, to: timeNumber.startIndex)
```

Comment:
the swift 3 string/collections API changed quite a bit, so could use some extra eyes on this...

/cc @theginbin